extract.py

```diff
@@ -46,6 +52,7 @@ def main():
         macroblock_data=args.include_macroblock_data,
         force=args.force,
         output_format=args.output_format,
+        logfile=args.data_log_file,
```

Comment:
Perhaps rename to `use_logfile` so it is clear that this is a switch, not the path to the logfile?